open_discussions/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'^$', index, name='open_discussions-index'),
     url(r'^auth_required/$', index),
     url(r'^content_policy/$', index),
+    url(  # so that we can use reverse() to link to this
```

Comment:
What were we doing before? Was it entirely handled on the frontend?

Reply:
Yeah, we basically wildcard matched all urls to the index route and handling it within the react app. Needed to add it here so I could use it in templates.